modules/openapi-generator/src/test/java/org/openapitools/codegen/DefaultCodegenTest.java

```diff
@@ -1309,10 +1323,9 @@ private void verifyMyPetsDiscriminator(CodegenDiscriminator discriminator) {
         test.setPropertyName(prop);
         test.setPropertyBaseName(prop);
         test.setMapping(null);
-        test.setMappedModels(new HashSet<CodegenDiscriminator.MappedModel>(){{
-            add(new CodegenDiscriminator.MappedModel("Cat", "Cat"));
-            add(new CodegenDiscriminator.MappedModel("Lizard", "Lizard"));
-        }});
+        test.setMappedModels(Set.of(
```

Comment:
need to move to jdk11...
https://github.com/OpenAPITools/openapi-generator/issues/5360

Reply:
Codebase uses `Sets.newHashSet` from guava in other places

Reply:
Also all these `Set.of` changes aren't related to the issue, so should land in a separate PR.